Tests/ComposableTests.swift

```diff
@@ -210,4 +210,26 @@ class ComposableTests: XCTestCase {
         XCTAssertEqual(["a", "b", "c"].joined() as NSAttributedString, "abc".styled(with: StringStyle()))
     }
 
+    func testKerningStrippingOnLastCharacter() {
+        let styleWithTracking = StringStyle(.tracking(.point(0.5)))
+
+        let testCases: [String] = [
```

Comment:
Let’s capture the line number of this, calculate the index of each test in the array, and pass it to `XCTAssertEqual` methods so that failures occur on the correct lines. This technique is used in a couple places in BonMot’s tests, and is outlined here: https://www.rightpoint.com/rlabs/2019/03/xctest-line-by-line/